logstash-codec-line.gemspec

```
@@ -21,6 +21,8 @@ Gem::Specification.new do |s|
 
   # Gem dependencies
   s.add_runtime_dependency "logstash-core-plugin-api", ">= 1.60", "<= 2.99"
+  s.add_runtime_dependency "logstash-mixin-ecs_compatibility_support", '~> 1.3', '< 2'
```

The `~>1.3` makes the `< 2` redundant.

```suggestion
  s.add_runtime_dependency "logstash-mixin-ecs_compatibility_support", '~> 1.3'
```